kubeflow/training-operator/gpt-neox/02-checkpoint-download-job.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: neox-checkpoint-download
+spec:
+  template:
+    spec:
+      containers:
+        - name: checkpoint-downloader
+          image: debian:11.6-slim
+          imagePullPolicy: IfNotPresent
+          command: [ "/bin/bash", "-c" ]
+          args:
+            - apt-get update && apt-get install -y wget;
+              mkdir -p /mnt/checkpoints/20B_checkpoints;
+              cd /mnt/checkpoints;
+              wget --cut-dirs=5 -nH -r --no-parent --reject "index.html*" https://the-eye.eu/public/AI/models/GPT-NeoX-20B/slim_weights/ -P 20B_checkpoints
```

Comment:
Are some of these things we can template? Like the URL? Not sure how this works for kubeflow.